ui/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "engines": {
-    "node": ">=16.0.0 <17.0.0"
+    "node": ">=16.0.0"
```

Comment:
Please revert this from this PR (it may be something we want to do -- but we need to evaluate it separately).